tests/integ/scala/test_dataframe_writer_suite.py

```diff
@@ -20,6 +20,11 @@
 from tests.utils import TestFiles, Utils
 
 
+@pytest.mark.xfail(
```

Comment:
This test should be refactored and enabled/skipped with a TODO, I think column_order is not supported.